### PR TITLE
Feature/Settings page 

### PR DESCRIPTION
**Pull request by RakeshUP**
## Description
Settings page with hardcoded data is added to the UI

Task: [8](https://aragonassociation.atlassian.net/jira/software/c/projects/APP/boards/41?modal=detail&selectedIssue=APP-8)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

### CODE REVIEW

File: packages/web-app/src/components/descriptionList/index.tsx [hunk 1]
```diff
@@ -17,16 +17,18 @@ export const DescriptionListContainer: React.FC<DescriptionListProps> = ({
...
-        <CheckboxSimple
-          label="These values are correct"
-          multiSelect
-          onClick={onChecked}
-        />
-      </div>
-    </HStack>
+    {onEditClick && onChecked && (
+      <HStack>
+        <ButtonText label="Edit" mode="ghost" onClick={onEditClick} />
```
Comment (Fabricevladimir) [inline packages/web-app/src/components/descriptionList/index.tsx]: Translation strings are calling out to you!

File: packages/web-app/src/pages/settings.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+import {PageWrapper} from 'components/wrappers';
+import {DescriptionListContainer, Dl, Dt, Dd} from 'components/descriptionList';
+import {useGlobalModalContext} from 'context/globalModals';
+
+const Settings: React.FC = () => {
+  const {t} = useTranslation();
+  const {open} = useGlobalModalContext();
+
+  return (
+    <PageWrapper title={'DAO Settings'}>
```
Comment (Fabricevladimir) [inline packages/web-app/src/pages/settings.tsx]: On the mobile version, there is a button on the PageWrapper header, so I think you can set the `buttonLabel` conditionally on screen size.

Also, there is a description on the mobile designs, which I guess we can use on desktop as well

File: packages/web-app/src/components/wrappers/pageWrappers.tsx [hunk 3]
```diff
@@ -10,8 +10,8 @@ import {useMappedBreadcrumbs} from 'hooks/useMappedBreadcrumbs';
 type ChangeSign = -1 | 0 | 1;
 
 export type PageWrapperProps = SectionWrapperProps & {
```
Comment (Fabricevladimir) [inline packages/web-app/src/components/wrappers/pageWrappers.tsx:18]: Would you mind updating to the `useMappedBreadcrumbs` hook to include the base path icon for settings?

File: packages/web-app/src/pages/settings.tsx [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+            <Dt>{t('labels.minimumSupport')}</Dt>
+            <Dd>50%</Dd>
+          </Dl>
+          <Dl>
+            <Dt>{t('labels.minimumDuration')}</Dt>
+            <Dd>5 Days 12 Hours 30 Minutes</Dd>
+          </Dl>
+        </DescriptionListContainer>
+
+        <ButtonText
```
Comment (Fabricevladimir) [inline packages/web-app/src/pages/settings.tsx:123]: On the mobile version, this button takes up the whole width

**Review by Fabricevladimir — COMMENTED** [review]
Thanks! Looking good. Left a couple of comments

**Review by WissenIstNacht — APPROVED** [review]
Hey Rakesh, sorry for takin so long with this.

Aside from Fabrices comments, everything looks good. Thanks a lot for the changes.

File: packages/web-app/src/components/descriptionList/index.tsx [hunk 5]
```diff
@@ -1,34 +1,45 @@
...
+      {onEditClick && onChecked && (
+        <HStack>
+          <ButtonText
+            label={t('labels.edit')}
+            mode="ghost"
+            onClick={onEditClick}
+          />
+          <div className="flex-shrink-0 tablet:w-3/4">
+            <CheckboxSimple
+              label="These values are correct"
```
Comment (Fabricevladimir) [inline packages/web-app/src/components/descriptionList/index.tsx:33]: Forgot this one!

**Review by Fabricevladimir — APPROVED** [review]
Thanks, Rakesh! Ready to go 🚀

**Pull request merged**